### PR TITLE
add utils/env package

### PR DESCRIPTION
**Pull request by zirain**
fixes: #334 

Signed-off-by: hejianpeng <hejianpeng2@huawei.com>

### CODE REVIEW

File: internal/utils/env/env.go [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package env
```
Comment (arkodg) [inline internal/utils/env/env.go:1]: nit: should we call the package `moreos` and API `LookupEnv` :) ?

File: internal/utils/env/env_test.go [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+	got = Lookup("TEST_ENV_KEY", d)
+	require.Equal(t, got, time.Second*10)
+
+	os.Clearenv()
+	got = Lookup("TEST_ENV_KEY", d)
+	require.Equal(t, got, d)
+
+	os.Setenv("TEST_ENV_KEY", "1axx")
+	got = Lookup("TEST_ENV_KEY", d)
+	require.Equal(t, got, d)
```
Comment (arkodg) [inline internal/utils/env/env_test.go:69]: shouldnt this be `"1axx"`

File: internal/utils/env/env_test.go [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+	require.Equal(t, got, 1000)
+
+	os.Clearenv()
+	got = Lookup("TEST_ENV_KEY", i)
+	require.Equal(t, got, i)
+
+	os.Clearenv()
+	os.Setenv("TEST_ENV_KEY", "10s")
+	got = Lookup("TEST_ENV_KEY", i)
+	require.Equal(t, got, i)
```
Comment (arkodg) [inline internal/utils/env/env_test.go:49]: shouldnt this be 10s ?

Comment (zirain) [reply internal/utils/env/env_test.go:49]: nop, `10s` is not a vaid `int`, backoff to defaultValue.

File: internal/utils/env/env_test.go (hunk 2 above)
Comment (zirain) [reply internal/utils/env/env_test.go:69]: same as above

File: internal/utils/env/env.go (hunk 1 above)
Comment (zirain) [reply internal/utils/env/env.go:1]: `moreos` sounds like handle the different between platform, like your guys do in [func-e](https://github.com/tetratelabs/func-e/tree/master/internal/moreos) project.

Comment (arkodg) [reply internal/utils/env/env.go:1]: cool, valid argument

File: internal/utils/env/env_test.go (hunk 3 above)
Comment (arkodg) [reply internal/utils/env/env_test.go:49]: ah thanks for clarifying that, logic makes sense

**Review by arkodg — APPROVED** [review]
LGTM, thanks for adding this util !

**Pull request merged**